internal/converter/internal/otelcolconvert/testdata/jaeger.alloy

```diff
@@ -5,11 +5,11 @@ otelcol.receiver.jaeger "default" {
 		thrift_http { }
 
 		thrift_binary {
-			max_packet_size = "63KiB488B"
+			max_packet_size = "65kB"
```

Comment:
Much better 😌 

However now I'm noticing that this would be a breaking change since the new package doesn't support this multi-suffix format, and this may affect users who used the converter and have this in their configuration files now. 

There's two things we could do here:

1. Support multi-suffix formats
2. Only use the new units package going forward for new fields

Reply:
I fixed this; the new package now also supports `63KiB488B`, though I wouldn't say we would recommend or document this.